src/_data/team.yml

```diff
@@ -520,7 +520,7 @@
   image: /assets/custom/img/team/chris_eyre.jpg
   role: Software Craftsman
   fullDescription:
-    <p>Chris has over 20 years of experience working in the software development across a range of industries. He has worked in Defence, Banking, Insurance, Futures Trading, Betting and Digital Publishing. </p>
+    <p>Software craftsman and <a href="https://leanpub.com/development2019">author</a> Chris has over 25 years of experience working in the software development across a range of industries. He has worked in Defence, Banking, Insurance, Futures Trading, Betting and Digital Publishing. </p>
```

Comment:
Maybe `experience working in the software development` -> `experience working in software development`.